distribution/tools/server-cli/src/main/java/org/elasticsearch/server/cli/JvmOptionsParser.java

```diff
@@ -70,25 +71,26 @@ SortedMap<Integer, String> invalidLines() {
      * variable.
      *
      * @param args            the start-up arguments
-     * @param configDir       the ES config dir
+     * @param processInfo     info to pass through to the new Java subprocess
      * @param tmpDir          the directory that should be passed to {@code -Djava.io.tmpdir}
-     * @param envOptions      the options passed through the ES_JAVA_OPTS env var
      * @return the list of options to put on the Java command line
-     * @throws InterruptedException if the java subprocess is interrupted
-     * @throws IOException          if there is a problem reading any of the files
+     * @throws RuntimeException     if the java subprocess is interrupted
```

Comment:
What advantage is there to wrapping an InterruptedException here?

Reply:
I tried to stay as close to the original code as possible; in the original code this method was always called within `ServerProcess#start`, which was wrapping all checked exception into runtime exceptions. 
Do you want to change it here and avoid wrapping them?

Reply:
Re-thinking about this: letting `InterruptedException` here and wrapping at the call site (`ServerCli#startProcess`) would be a little bit more cumbersome, but closer to the original code; I'll change it in that direction.